packages/api/src/services/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as Blocks } from "./blocks";
+export { default as Emps } from "./emp-state";
+export { default as Registry } from "./emp-registry";
+export { default as Prices } from "./prices";
```

Comment:
nit: not a strong opinion, but curious why you chose to export these as defaults, and then name them a level up?

Another way you could do this is to give it a name at the implementation level:
```ts
export function SomeName = ...;
```

And then at the next level up:
```ts
export * from "./ModuleName";
```

or if you want to be specific:
```ts
export { SomeName as SomeOtherName } from "ModuleName";
```

I'm guessing you're aware of all this syntax, but curious why you chose to do default exports and then un-default them at the next level. Is that helpful for some specific use case? Or is it more readable?

Reply:
yea its a great question, we talked offline but ill summarize:
- i see naming as separate problem from the implementation in these cases, the writer can just export a single function in most and pass naming onto a higher level process. 
- this creates a nice predictable template when adding new "modules", its almost to the point where you can copy a folder from a past implementation, and replace the logic with your own. no need to find and replace specific module names within the code.
- i like that using the default puts the focus on what's being exported as the main subject, and implies there's no need to export any other named things.

Reply:
makes sense!